src/common/snippets/src/op/perf_count.cpp

```diff
@@ -3,10 +3,117 @@
 //
 #ifdef SNIPPETS_DEBUG_CAPS
 
+#include <fstream>
+
 #include "snippets/op/perf_count.hpp"
 
 namespace ov {
 namespace snippets {
+
+//////////////////utils///////////////
+
+namespace utils {
+
+//////////////////utils::Dumper///////////////
+
+void Dumper::init(const std::string &params) {
+    m_params = params;
+}
+
+//////////////////utils::ConsoleDumper///////////////
+
+ConsoleDumper::~ConsoleDumper() {
+    OPENVINO_ASSERT(m_accumulation.size() == m_iteration.size(),
+                    "accumulation size should be the same as iteration size in perf_count_end node.");
+    auto iterator_iter = m_iteration.begin();
+    auto iterator_acc = m_accumulation.begin();
+    uint64_t avg_max = 0;
+    for (; iterator_iter != m_iteration.end(); ++iterator_iter, ++iterator_acc) {
+        const auto iter = *iterator_iter;
+        const auto acc = *iterator_acc;
+        uint64_t avg = iter == 0 ? 0 : acc / iter;
+        if (avg > avg_max)
+            avg_max = avg;
+    }
+
+    // max time of all threads: combine for reduce max
+    auto BinaryFunc = [](const uint64_t& a, const uint64_t& b) {
+        return a >= b ? a : b;
+    };
+
+    // max accumulation
+    uint64_t acc_max = m_accumulation.combine(BinaryFunc);
+    std::cout << "max accumulated time:" << acc_max << "ns" << std::endl;
+    // max avg
+    std::cout << "max avg time:" << avg_max << "ns" << std::endl;
+}
+
+void ConsoleDumper::update(const op::PerfCountEnd* node) {
+    auto accumulation = node->get_accumulation();
+    auto iteration = node->get_iteration();
+    OPENVINO_ASSERT(accumulation.size() == iteration.size(),
+                    "accumulation size should be the same as iteration size in perf_count_end node.");
+    auto iterator_iter = iteration.begin();
+    auto iterator_acc = accumulation.begin();
+    for (; iterator_iter != iteration.end(); ++iterator_iter, ++iterator_acc) {
+        m_accumulation.local() += *iterator_acc;
+        m_iteration.local() += *iterator_iter;
+    }
+}
+
+//////////////////utils::CSVDumper///////////////
+
+CSVDumper::CSVDumper(const std::string& csv_path) : csv_path(csv_path) {}
+
+CSVDumper::~CSVDumper() {
+    dump_brgemm_params_to_csv();
```

Comment:
It looks like the `dump_brgemm_params_to_csv` method is redundant, please remove.

Reply:
Agree, removed